tests/features/paragraphs/latest_blog_posts_paragraphs.feature

```diff
@@ -0,0 +1,61 @@
+@openy @api
+Feature: Latest Blogs Paragraphs
+  Check that Latest Blog Posts paragraphs output other nodes based on some rules
+
+  Background: Log in
+    Given I am logged in as a user with the "Administrator" role
+
+  Scenario: Paste latest blog post
+    And I go to "/node/add/landing_page"
+    And I fill in "Title" with "Landing"
+    And I select "One Column" from "Layout"
+    When I press "Add Latest blog posts"
+    And I press "Save and publish"
+    Then I should see "Mood-boosting foods"
+    And I should see "Start training now for the Y Run"
+    And I should see "Why you should give walking a try"
+    And I should see "Don’t let summer sneak up on your family"
+    And I should see "Add a water workout to your routine"
+    And I should see "Grown-up fun in Adult Sports Leagues"
+    When I click "Load More"
+    Then I should see "Nourish yourself with healthy fats"
+    And I should see "Cheddar-Cannelini Fondue"
+    And I should see "Mango-Avocado Salsa"
+
+  Scenario: Paste latest blog post (branch)
+    When I go to "node/1/edit"
```

Comment:
Why you have hardcoded id 1? According to our converstation, we shouldn't rely on demo content.